### PR TITLE
Add ability to return 422 for validation errors

### PR DESCRIPTION
**Pull request by GDSNewt**
This will help us see where users are hitting validation errors, and if we're seeing elevated numbers of errors. This'll help us debug and investigate issues. 

https://trello.com/c/n31BWiHm/203-return-a-422-for-validation-errors-in-business-form

### CODE REVIEW

**Review by brucebolt — APPROVED** [review]
Looks good 👍.

Could you add a link to the Trello card in the PR description?  It'll then appear in the Trello card automatically, which makes it easier to track which code change belongs to specific pieces of work.

**Comment by GDSNewt** [comment]
> Looks good 👍.
> 
> Could you add a link to the Trello card in the PR description? It'll then appear in the Trello card automatically, which makes it easier to track which code change belongs to specific pieces of work.

Thanks Bruce, I think I was adding it as you wrote this 😄

File: spec/controllers/coronavirus_form/business_details_controller_spec.rb [hunk 1]
```diff
@@ -83,8 +83,7 @@
         company_location: I18n.t("coronavirus_form.questions.business_details.company_location.options.european_union.label"),
       ).except(:company_postcode)
 
-      expect(response).to have_http_status(:unprocessable_entity)
-      expect(response).to render_template(current_template)
+      expect(response).to redirect_to(contact_details_path)
```
Comment (brucebolt) [inline spec/controllers/coronavirus_form/business_details_controller_spec.rb]: This merge commit seems to have removed the `expect(response).to have_http_status(:unprocessable_entity)` that you made in a previous commit

Comment (GDSNewt) [reply spec/controllers/coronavirus_form/business_details_controller_spec.rb]: Hi Bruce, that was intentional. I thought if we were expecting a redirect then we should not be expecting a 422 error. At least i think that's what it's supposed to be doing 🤔

**Pull request merged**